sos/report/plugins/openstack_tripleo.py

```diff
@@ -19,16 +19,18 @@ class OpenStackTripleO(Plugin, IndependentPlugin):
     plugin_name = 'openstack_tripleo'
     profiles = ('openstack', 'openstack_controller', 'openstack_compute')
     packages = ('openstack-selinux',)
+    tripleo_log_paths = [
+        '/var/log/paunch.log',
+        '/var/lib/container-puppet/',
+        '/var/lib/tripleo-config/',
+        '/var/lib/tripleo/',
+        '/etc/puppet/hieradata/'
+    ]
```

Comment:
Pulling the paths into a list like this is fine, however stylistically I would prefer `self.tripleo_log_paths` be set inside `setup()` - a handful of plugins do this today and we should remain consistent.

Reply:
done.